diff --git a/lib/git.zsh b/lib/git.zsh
index 640561e..d7fafa1 100644
--- a/lib/git.zsh
+++ b/lib/git.zsh
@@ -17,7 +17,10 @@ function parse_git_dirty() {
     if [[ "$DISABLE_UNTRACKED_FILES_DIRTY" == "true" ]]; then
       FLAGS+='--untracked-files=no'
     fi
-    STATUS=$(command git status ${FLAGS} 2> /dev/null | tail -n1)
+    if [[ "$(pwd)" != *"${HOME}/Developer/electron-gn/src"*  &&
+    "$(pwd)" != *"${HOME}/Developer/chromium/src"* ]]; then
+      STATUS=$(command git status ${FLAGS} 2> /dev/null | tail -n1)
+    fi
   fi
   if [[ -n $STATUS ]]; then
     echo "$ZSH_THEME_GIT_PROMPT_DIRTY"
